docs/framework/unmanaged-api/debugging/ixclrdataprocess-endenummethodinstancesbyaddress-method.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "IXCLRDataProcess::EndEnumMethodInstancesByAddress Method"
+ms.date: "01/15/2019"
+api.name:
+  - "IXCLRDataProcess::EndEnumMethodInstancesByAddress"
+api.location:
+  - "mscordacwks.dll"
+api.type:
+  - "COM"
+f1.keywords:
+  - "IXCLRDataProcess::EndEnumMethodInstancesByAddress Method"
+helpviewer.keywords:
+  - "IXCLRDataProcess::EndEnumMethodInstancesByAddress Method [.NET Framework debugging]"
+topic_type:
+  - "apiref"
+author: "cshung"
+ms.author: "andrewau"
+---
+# IXCLRDataProcess::EndEnumMethodInstancesByAddress Method
+End enumerate method instances of this process by an address.
```

Comment:
Ends enumerating method instances of this process by address -- ?
The definition is unclear.